core/domain/stats_jobs_one_off.py

```diff
@@ -141,14 +141,17 @@ def map(item):
                 else:
                     try:
                         exploration = exp_services.get_exploration_by_id(
-                            item.exploration_id, item.exploration_version)
+                            item.exploration_id,
+                            version=item.exploration_version)
                     except Exception:
                         # Exploration does not exist.
                         return
-                    dest_state = exploration.states[
-                        unicode_state_name].interaction.default_outcome.dest
-                    if dest_state != unicode_state_name:
-                        useful_feedback_count += 1
+
+                    state = exploration.states[unicode_state_name]
+                    if state.interaction.default_outcome:
```

Comment:
Also, what do you want to do in the else case? Should at least yield an error so we can track it. Not sure whether to additionally return or not.

Reply:
Done.